CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## unreleased
+
+* Set client `client.overlay.runtimeErrors` to `false` in webpack devServer configuration to avoid showing errors at window resize.
+
```

Comment:
- Please add the category '### serve' as below(because this is a change specified in 'enact serve')

- Please start with 'Added', 'Fixed', 'Updated', etc. as below.

- Please write what you did, not how you did it as below.
You don't have to write down in detail how you did it.